cli/pcluster/config/validators.py

```diff
@@ -508,12 +510,12 @@ def ec2_iam_role_validator(param_key, param_value, pcluster_config):
         iam = boto3.client("iam")
         arn = iam.get_role(RoleName=param_value).get("Role").get("Arn")
         account_id = boto3.client("sts", endpoint_url=_get_sts_endpoint()).get_caller_identity().get("Account")
-
-        iam_policy = _get_pcluster_user_policy(get_partition(), get_region(), account_id)
+        scheduler = pcluster_config.get_section("cluster").get_param_value("scheduler")
+        iam_policy = _get_pcluster_instance_policy(scheduler, get_partition(), get_region(), account_id)
```

Comment:
this is just a naming change right? we were always validating instance policies, and the function that did so was just not accurately named?

i'm pretty certain this is true looking at the code, but wanted to double-check